lib/msf/base/serializer/readable_text.rb

```diff
@@ -569,68 +569,82 @@ def self.dump_generic_module(mod, indent = '')
   # @param missing [Boolean] dump only empty required options.
   # @return [String] the string form of the information.
   def self.dump_options(mod, indent = '', missing = false)
```

Comment:
I only added these changes here and not to advanced/evasion options since only the regular options are using it at the minute and there's already a lot of overlap in dumping out of each option type and I think it's worth looking into consolidating them and passing in whether you want regular/advanced/evasion options dumped